Kinvey/KinveyTests/PerformanceProductTestCase.swift

```diff
@@ -0,0 +1,447 @@
+//
+//  PerformanceProductTestCase.swift
+//  Kinvey
+//
+//  Created by Victor Hugo on 2017-02-28.
+//  Copyright © 2017 Kinvey. All rights reserved.
+//
+
+import XCTest
+import RealmSwift
+@testable import Kinvey
+import PromiseKit
+
+class Product: Entity {
+    
+    let materialNumber = RealmOptional<Int>()
+    dynamic var productDescription: String?
+    dynamic var vertical: String?
+    dynamic var subVertical: String?
+    dynamic var classification: String?
+    dynamic var materialGroup: String?
+    let materialFreightGroup = RealmOptional<Int>()
+    dynamic var materialType: String?
+    dynamic var baseUOM: String?
+    let packageQuantity = RealmOptional<Int>()
+    dynamic var gsaFlag: String?
+    dynamic var discontinuedIndicator: String?
+    dynamic var deactivateDate: String?
+    dynamic var taaFlag: String?
+    let minOrderQuantity = RealmOptional<Int>()
+    let active = RealmOptional<Bool>()
+    dynamic var created: String?
+    dynamic var modified: String?
+    
+    override static func collectionName() -> String {
+        return "Product"
+    }
+    
+    override func propertyMapping(_ map: Map) {
+        super.propertyMapping(map)
+        
+        materialNumber <- ("materialNumber", map["MaterialNumber"])
+        productDescription <- ("productDescription", map["Description"])
+        vertical <- ("vertical", map["Vertical"])
+        subVertical <- ("subVertical", map["SubVertical"])
+        classification <- ("classification", map["Classification"])
+        materialGroup <- ("materialGroup", map["MaterialGroup"])
+        materialFreightGroup <- ("materialFreightGroup", map["MaterialFreightGroup"])
+        materialType <- ("materialType", map["MaterialType"])
+        baseUOM <- ("baseUOM", map["BaseUOM"])
+        packageQuantity <- ("packageQuantity", map["PackageQuantity"])
+        gsaFlag <- ("gsaFlag", map["GSAFlag"])
+        discontinuedIndicator <- ("discontinuedIndicator", map["DiscontinuedIndicator"])
+        deactivateDate <- ("deactivateDate", map["DeactivateDate"])
+        taaFlag <- ("taaFlag", map["TAAFlag"])
+        minOrderQuantity <- ("minOrderQuantity", map["MinOrderQuantity"])
+        active <- ("active", map["Active"])
+        created <- ("created", map["Created"])
+        modified <- ("modified", map["Modified"])
+    }
+    
+}
+
+class PerformanceProductTestCase: KinveyTestCase {
+    
+    var productsJsonArray: [JsonDictionary]?
+    
+    lazy var fileStore = FileStore.getInstance()
+    
+    lazy var productsJsonFile: File? = {
```

Comment:
right... I'm already doing this... the file is actually stored in a Kinvey backend (as a file) and the unit test is also disabled by default so Travis will not run.